relayer-cli/src/conclude.rs

```diff
@@ -0,0 +1,110 @@
+//! Custom-made solution to output a JSON return message and ensure a return code
+//! from a CLI command. The main use-case for this module is to provide a consistent output for
+//! queries and transactions.
+//!
+//! ## Examples:
+//!
+//! - Exit from a query/tx with a string error:
+//!
+//! ```ignore
```

Comment:
why are these ignored?

Reply:
Ignored because it's not really testing anything, it's just code snippets to help the developer use the module in the future. There's nothing to assert about this code: The command `Output::with_...exit()` writes to stdout and exits the program, but I don't know how to assert that the exit code from the program was `0` or `1`, nor how to assert that the program wrote anything to stdout.

If it is seems valuable, I can propose two options to make this un-ignored:

1. replace the `ignore` attribute with [`no_run`](https://doc.rust-lang.org/rustdoc/documentation-tests.html#attributes). This means that the code snippet is expected to compile (but not run).

2. remove the `ignore` attribute and try to have full-fledged doc tests that compile+run, but it's unclear if there will by any asserts, actual tests.

Let me know what you think.

Reply:
I like the `no_run` option but it's up to you.

Reply:
In the end, I went with attribute `compile_fail` and added an explanation on why the doctests will not compile (the reason is that the doctests need access to the `Output` struct, but this is crate-private and hence not accessible within doc tests).